Bugzilla/Report/SecurityRisk.pm

```diff
@@ -146,6 +150,65 @@ sub _build_products {
   return \@products;
 }
 
+sub _build_missing_products {
+  my ($self) = @_;
+  my $dbh = Bugzilla->dbh;
+  my $products = join ', ', map { $dbh->quote($_) } @{$self->products};
+  my $query = qq{
+        SELECT
+            name
+        FROM
+            products
+         WHERE
+            products.name IN ($products)
```

Comment:
Need to use $dbh->sql_in() here, for portability.

Note you can embed code in a string literal with:
```perl
"@{[ ]}"
```

which is sort of a hack but one I've used a few times recently.

Reply:
Fixed! (Note to self, the `[ ]` inside is really necessary!)

Reply:
(really fixed now)